src/test/objectstore/allocator_replay_test.cc

```diff
@@ -458,6 +458,73 @@ int export_as_binary(char* fname, char* target_fname)
   return r;
 }
 
+int check_duplicates(char* fname)
+{
```

Comment:
I have my doubts about this checking procedure.
We build a set of regions that are marked "free" twice, using all replay data.
Then we check which other regions contributed to collision, but we look only into regions defined before first collision.
This would make sense if we wanted to have a clear picture: "Lets find who is in collision with FIRST error", but for that we should only capture ONE region into `invalid_extentsB`.

Reply:
yeah, non-perfect implementation indeed, reworking

Reply:
done, @aclamk please take a look